src/bridge/gasutils.ts

```diff
@@ -13,46 +16,31 @@ export namespace GasUtils {
 
     const makeGwei = (n: string): BigNumber => parseUnits(n, "gwei")
 
-    const ETH_GAS_PARAMS: GasParams = {
-        maxPriorityFee: makeGwei("1.5"),
-        bridgeGasLimit: BigNumber.from(100000)
-    };
-
-    const BOBA_GAS_PARAMS: GasParams = {
-        gasPrice:        makeGwei("10"),
-        approveGasLimit: BigNumber.from(60000),
-    };
-
-    const ARBITRUM_GAS_PARAMS: GasParams = {
-        gasPrice:       makeGwei("2.5"),
-        bridgeGasLimit: BigNumber.from(1500000),
-    };
-
-    const AVALANCHE_GAS_PARAMS: GasParams = {
-        gasPrice:        makeGwei("150"),
-        bridgeGasLimit:  BigNumber.from(800000),
-        approveGasLimit: BigNumber.from(75000),
-    }
-
-    const AURORA_GAS_PARAMS: GasParams = {
-        gasPrice: makeGwei('0'),
+    const CHAIN_GAS_PARAMS: ChainIdTypeMap<GasParams> = {
+        [ChainId.ETH]: {
+            maxPriorityFee: makeGwei("1.5"),
+            bridgeGasLimit: BigNumber.from(100000)
+        },
+        [ChainId.BOBA]: {
+            gasPrice:        makeGwei("10"),
+            approveGasLimit: BigNumber.from(60000),
+        },
+        [ChainId.ARBITRUM]: {
+            gasPrice:       makeGwei("2.5"),
+            bridgeGasLimit: BigNumber.from(1500000),
+        },
+        [ChainId.AVALANCHE]: {
+            gasPrice:        makeGwei("150"),
+            bridgeGasLimit:  BigNumber.from(800000),
+            approveGasLimit: BigNumber.from(75000),
+        },
+        [ChainId.AURORA]: {
+            gasPrice: makeGwei('0'),
```

Comment:
Gas price on Aurora is no longer set to 0: https://twitter.com/AlexAuroraDev/status/1490353255817302016

Reply:
Thank you sir